termbox/compat.go

```diff
@@ -357,3 +381,17 @@ type Cell struct {
 	Fg Attribute
 	Bg Attribute
 }
+
+const (
```

Comment:
I haven't checked, but are these names exposed by termbox?

Reply:
I only added what was necessary to get a build. So yes.